app/src/main/java/com/jjswigut/matters/ui/EditMatterFragment.kt

```diff
@@ -146,5 +145,12 @@ class EditMatterFragment : BaseFragment() {
         super.onDestroyView()
         _binding = null
     }
+
+    companion object {
+        private const val ARG_MATTER = "arg_matter"
+        fun newBundle(matter: Matter) = Bundle().apply {
```

Comment:
so this is how we create a bundle to navigate to this fragment. you can see in MatterListfragment how it is used - but essentially when you call this function with a matter object, you will get a bundle back containing that matter object at the key ARG_MATTER

the syntax might look a little funky if you haven't seen it before, but it's 100% equivalent to just doing

`kotlin
fun newBundle(matter: matter) : Bundle {
    val bundle = Bundle()
    bundle.putSerializable(ARG_MATTER, matter)
    return bundle
}
```

this is just a little kotlin syntax sugar in action for us